tests/production/test_pvnet.py

```diff
@@ -0,0 +1,27 @@
+from datetime import datetime
+
+from freezegun import freeze_time
+from torchdata.datapipes.iter import IterableWrapper
+
+from ocf_datapipes.training.pvnet import (
+    construct_sliced_data_pipeline,
+)
+from ocf_datapipes.utils.consts import Location
+
+
+@freeze_time("2020-04-02 02:30:00")
```

Comment:
I don't know this function. But from what I can tell, the time here is "2020-04-02 02:30:00" and the time in the `t0_pipe` is "2020-04-02 00:30:00". Is this right?

Reply:
yea. the freeze time mocks things like datetime.now() to get that time. Its useful when trying to test those function out